ocaml/xapi/storage_smapiv1_wrapper.ml

```diff
@@ -82,7 +82,7 @@ let s_of_vm = Storage_interface.Vm.string_of
 
 let vm_of_s = Storage_interface.Vm.of_string
 
-module D = Debug.Make (struct let name = "storage_impl" end)
+module D = Debug.Make (struct let name = "storage_smapiv1_wrapper" end)
```

Comment:
```suggestion
module D = Debug.Make (struct let name = __MODULE__ end)
```

Reply:
Ah, small difference here: `__MODULE__` capitalises the name, so it's Storage_smapiv1_wrapper in the logs. Not a big deal if you know what to look for...